src/pay/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const GENERAL_PAY_ERROR_MESSAGE = 'PAY_ERROR';
```

Comment:
I am leaning towards consolidate this into `src/api/constants`.

Reply:
I think that's fine, though I do worry about that file growing large over time.

Were you picturing component error messages being in a single constants file as well per domain?

Reply:
I think many error messages are going to be the exact same between APIs, so having them in one place it will make for us easier to oversee the entire dev experience, and help those API converge into similar language.